tests/examples/other/localhost-links-different-port.html

```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+</head>
+
+<body>
+  <a href="http://localhost:55555">localhost link on a different port</a>
+  <a href="index.html">html</a>
```

Comment:
remove `<a href="index.html">html</a>` and `404.html` and `names()` should become `/${source}/localhost-links-different-port.html`